mavros/src/mavros/ftp.py

```diff
@@ -53,6 +53,9 @@ def __init__(self, name, mode):
     def __del__(self):
         self.close()
 
+    def __exit__(self):
+        self.close()
+
```

Comment:
Class already have `__exit__()` method, look bottom.

Reply:
It's true. I was not careful